@navikt/core/react/src/accordion/Accordion.tsx

```diff
@@ -31,15 +32,9 @@ interface AccordionComponent
 
 export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
-   * @deprecated "default" will be the only variant.
-   * @default "default"
+   * @deprecated Removed in v8, but backwards compatible. Use `data-color` instead.
```

Comment:
Not really removed then 😄

```suggestion
   * @deprecated Use `data-color` instead.
```

```suggestion
   * @deprecated Will be removed in a future major version. Use `data-color` instead.
```